airflow/providers/microsoft/azure/hooks/asb_message.py

```
@@ -0,0 +1,122 @@
+        extras = conn.extra_dejson
+
+        self.connection_string = str(
+            extras.get('connection_string') or extras.get('extra__azure_service_bus__connection_string')
+        )
+
+        return ServiceBusClient.from_connection_string(conn_str=self.connection_string, logging_enable=True)
+
+    def send_message(
+        self, queue_name: str, messages: Union[str, List[str]], batch_message_flag: bool = False
+    ):
+        """
+        By using ServiceBusClient Send message(s) to a Service Bus Queue. By using
+        batch_message_flag it enables and send message as batch message
+
+        :param queue_name: The name of the queue or a QueueProperties with name.
+        :param messages: Message which needs to be sent to the queue. It can be string or list of string.
+        :param batch_message_flag: bool flag, can be set to True if message needs to be sent as batch message.
+        """
+        if queue_name is None:
+            raise AirflowBadRequest("Queue name cannot be None.")
+        if not messages:
+            raise AirflowException("Message is empty.")
+        service_bus_client = self.get_conn()
+        try:
+            with service_bus_client:
+                sender = service_bus_client.get_queue_sender(queue_name=queue_name)
+                with sender:
+                    if isinstance(messages, str):
+                        if not batch_message_flag:
+                            msg = ServiceBusMessage(messages)
+                            sender.send_messages(msg)
+                        else:
+                            self.send_batch_message(sender, [messages])
+                    else:
+                        if not batch_message_flag:
+                            self.send_list_messages(sender, messages)
+                        else:
+                            self.send_batch_message(sender, messages)
+        except Exception as e:
+            raise AirflowException(e)
```

There is no need for this to be an AirflowException -- so lets git rid of the try/except block and just let the exception be thrown.

---

Removed try/except block and AirflowException